pkgs/sagemath-standard/constraints_pkgs.txt

```diff
@@ -0,0 +1,20 @@
+# This "constraints file" can be used for forcing pip
+# (and any tools that delegate to pip, such as pypa/build)
+# to install the distribution packages included in
+# the SageMath monorepository only from their source trees
+# in SAGE_ROOT/pkgs/ instead of from PyPI.
+#
+# Example: Building a sagemath-standard wheel
+#
+# [alice@localhost sage]$ ./bootstrap
+# [alice@localhost sage]$ ./configure
+# [alice@localhost sage]$ export MAKE="make -j16" SAGE_NUM_THREADS=16
+# [alice@localhost sage]$ make all-sage-local
+# [alice@localhost sage]$ export PIP_CONSTRAINT="$(pwd)/pkgs/sagemath-standard/constraints_pkgs.txt"
+# [alice@localhost sage]$ ./sage -sh -c 'python3 -m build -v -v pkgs/sagemath-standard'
+#
```

Comment:
It seems that this comment is copied and slightly modified from constraints_pkgs.txt. But It is not right because this constraints file is only for sagemath-standrad.

Reply:
No, these instructions are also correct. Both `constraints_pkgs.txt` files can be used to correctly build pkgs/sagemath-standard

Reply:
I meant the lines L1 ~ L7.

Reply:
Oh. Rephrased in 368e9f6, please take a look if that's clearer

Reply:
Thanks. 

I think current L8 is redundant or just "Example:" suffices, but you may leave it if you think otherwise.